src/platform/i386/kernel.c

```diff
@@ -138,6 +147,9 @@ kmain(struct multiboot *mboot, u32_t mboot_magic, u32_t esp)
        	comp_init();
```

Comment:
I think it has been like that from before. I've not touched that part, there is no diff of base to my version at these lines. I'll fix it anyway in my next pull request. :)